include/freerdp/freerdp.h

```diff
@@ -360,7 +360,14 @@ extern "C"
 		Disconnect_Ultimatum_channel_purged = 4
 	};
 
+#ifdef __cplusplus
```

Comment:
useless change?

Reply:
The include of some glib headers errored out here complaining about templates under C linkage deep inside the include chain... But that might go away with the Wrapper struct as well.